carla/recourse_methods/processing/counterfactuals.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+import numpy as np
+import pandas as pd
+
+from carla.models.api import MLModel
+
+
+def counterfactual_to_dataframe(
+    mlmodel: MLModel, counterfactuals: List
+) -> pd.DataFrame:
+    df_cfs = pd.DataFrame(
+        np.array(counterfactuals), columns=mlmodel.feature_input_order
+    )
+    df_cfs[mlmodel.data.target] = np.argmax(mlmodel.predict_proba(df_cfs), axis=1)
```

Comment:
Nitpick: `counterfactual_to_dataframe` implies that there is merely some datatype conversion going on, while we also do some predicting and cleaning. Maybe we can find a better name for the function

Reply:
Right, this could be very confusing, I'll change it to `check_counterfactuals()`